app/assets/javascripts/retro.js

```diff
@@ -1,20 +1,18 @@
-$(document).ready(function() {
+$(document).ready(() => {
```

Comment:
There's a conflict between `turbolinks` gem and `$(document).ready()` event.
[This](https://stackoverflow.com/questions/36110789/rails-5-how-to-use-document-ready-with-turbo-links) can help 👍